dockstore-integration-testing/src/test/java/io/dockstore/webservice/WebhookIT.java

```diff
@@ -965,6 +965,26 @@ public void testSnapshotAppTool() throws Exception {
         assertEquals(1, imageCount);
     }
 
+    @Test
+    public void testChangingAppToolTopics() throws Exception {
+        CommonTestUtilities.cleanStatePrivate2(SUPPORT, false);
+        final ApiClient webClient = getWebClient(BasicIT.USER_2_USERNAME, testingPostgres);
+        final io.dockstore.openapi.client.ApiClient openApiClient = getOpenAPIWebClient(BasicIT.USER_2_USERNAME, testingPostgres);
+        io.dockstore.openapi.client.api.UsersApi usersApi = new io.dockstore.openapi.client.api.UsersApi(openApiClient);
+        WorkflowsApi client = new WorkflowsApi(webClient);
```

Comment:
Could use `openApiClient` to create an openapi `WorkflowsApi` instead of creating a new swagger client on line 971

Reply:
Ran into https://ucsc-cgl.atlassian.net/browse/SEAB-4046, will need to investigate separately I think